components/Map/Map.module.css

```diff
@@ -1,4 +1,9 @@
 .map_container {
   height: 450px;
-  width: 100%;
+  width: 85%;
+  margin-left: 100px;
+  margin-right: 100px;
+  position: absolute;
```

Comment:
Ich würde vermeiden `position: absolute` für die meisten Elemente zu nutzen. `position: absolute` eignet sich meist nur, wenn du "Dekorations-Elemente" genau positionieren möchtest (z.B. ein Sales-Badge an einem Produktfoto oder ein paar Balken zur Deko im Header). Alle Elemente mit Content würde ich nicht mit `position: absolute` positionieren. Hier ist ein Beitrag zu der `position` Property: 
https://www.freecodecamp.org/news/css-positioning-position-absolute-and-relative/

In diesem Fall möchtest du sicherlich nur einen Abstand nach oben ergänzen oder? Dafür könntest du `margin-top: 250px` nutzen.

Vielleicht kannst du ja beim `map_container` und `footer` schauen, ob du auch ohne `position: absolute` weiterkommst.